dev/Soniyanayak51/misclassificationVisualization/misclassificationVisualisationFunctions.py

```diff
@@ -0,0 +1,51 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import sklearn as skl
+from sklearn.model_selection import train_test_split
+import matplotlib.pyplot as plt
+from sklearn.utils import shuffle
+from sklearn.metrics import confusion_matrix
+import seaborn as sns
+import matplotlib.gridspec as gridspec
+from sklearn.preprocessing import StandardScaler
+from sklearn.manifold import TSNE
+from pandas import set_option
+from sklearn.svm import SVC
+from sklearn.linear_model import LogisticRegression
+from sklearn.datasets import load_iris
+
+    
+def misclassificationGraphs(y_test, preds):
+    '''
+    Plots a graph to show default values
+    Agrs: The label y
+    '''
+    predsSeries = pd.Series(preds)
+    predsSeries.reindex(index=y_test.index)
+    df_miss = pd.concat([y_test, predsSeries], axis=1, join='inner')
+    df_miss = df_miss.rename(columns={"default payment next month": "y_test", 0: "predsSeries"})
+    ax1 = sns.countplot(x="predsSeries", hue="y_test", data=df_miss, palette="Blues")
+ #   plt.show()
+    
+def plotGrapthForAttributes(df):
```

Comment:
I believe this is already included in your other module, in which case you can drop it from here.

Reply:
Removed this.